storage/innobase/buf/buf0lru.cc

```diff
@@ -450,8 +450,10 @@ buf_block_t *buf_LRU_get_free_block(bool have_mutex)
       buf_pool.page_cleaner_wakeup(true);
     mysql_mutex_unlock(&buf_pool.flush_list_mutex);
     if (my_cond_timedwait(&buf_pool.done_free, &buf_pool.mutex.m_mutex,
-                          &abstime))
+                          &abstime)) {
       buf_pool.LRU_warn();
```

Comment:
Monty wants do deviate from the K&R style and insert a line break before `{` except in a `switch` statement.

Reply:
right, I should have followed the convention in existing code. 
Corrected.